array_schema_evolution.go

```diff
@@ -0,0 +1,111 @@
+//go:build experimental
+// +build experimental
+
+/**
+ * @file   array_schema_experimental.go
+ *
+ * @section DESCRIPTION
```

Comment:
These Javadoc-like tags are not needed.

Also, traditional Go style is not to include the intervening `*`s in multiline comments.